examples/xor-mlp/project.clj

```diff
@@ -0,0 +1,6 @@
+(defproject learn-cortex "0.0.1-SNAPSHOT"
```

Comment:
Having project name here match the folder name is typical. Any reason for the difference here?